content/articles/an-overview-of-web-3-0/index.md

```diff
@@ -0,0 +1,105 @@
+The World wide web is a crucial element of our modern life. It has become part of us, and it would not be easy to outgrow our lives without it. This is because technology has greatly advanced as people can efficiently work remotely and easily communicate with the help of the internet. The reliance on the world wide web has grown in the last three decades and has gone through stages of evolution from [Web 1.O](https://websitebuilders.com/how-to/glossary/web1/), [Web 2.O](https://en.wikipedia.org/wiki/Web_2.0), and [Web 3.O](https://web3.foundation/).
+
+Most people are probably wondering what the future web will be like. Well, Web 3.O (Web 3) is the future web. Many people are confused about what the next generation internet (Web 3) will be like. Web 3 is a part of a much larger phase of transition that we will go through. Therefore, there is a more significant change going on right now on the internet, and lots of improvements are being deployed.
+
+The internet just got a lot more complicated. There's an increased awareness of web 3, and there is so much jargon in the space, and it isn't easy to understand what it's all about. This article will break down all the jargon for you and make you perfectly understand what web 3.O is all about.
+
+This article will explain Web 3 or the future of the web. If you want to stay up-to-date on how we'll change our lives soon, make sure you read this article to get the complete picture. Without further ado, let's get started.
+
+### Table of Contents
+[The Evolution of the Web](#the-evolution-of-the-web)
+[Why we need Web 3](#why-we-need-web-3)
+[How blockchain works](#how-blockchain-works)
+[Blockchain to make Web 3 possible](#blockchain-to-make-web-3-possible)
+
+### The Evolution of the Web
+The internet and web technology have certainly evolved in the past few years. The world wide web has evolved through different phases: Web 1.O, Web 2.O, and Web 3.O. Each phase has brought about new technologies, tools, and techniques relevant to recruiters.
+
+![Evolution of web](/engineering-education/an-overview-of-web-3-o/web-evolution.jpg)
+
+[Image source](https://martech.zone/web-1-web-2-web-3-0/)
+
+Let's dive deeper to find out how the internet evolved from its initiation to what we know it as today.
+
+#### What is Web 1.O
+Web 1.O was the first phase of the web revolution, also referred to as read-only web. It is primarily categorized by static websites that don't do much but display information on the page. You can read text, click different links, and write and send emails but can't attach or upload images and documents.
```

Comment:
You should be using a transition on this paragraph, i'e,   "For example,you can read text" to something like. Keep your sentences as simple as possible.